azure-pipelines.yml

```diff
@@ -782,8 +782,10 @@ jobs:
           PR_NUM: $(pr.num)
           PR_BRANCH: $(pr.branch)
 
-  - job: notify_user
-    condition: and(eq(variables['Build.Reason'], 'PullRequest'), not(canceled()))
+  - job: notify_release_pr
```

Comment:
This will also trigger for manually opened release PRs right? I guess there is no harm in it but it also seems a bit redundant.

Reply:
Good point. I've updated to condition to match the branch name created by the automated release PRs. It's obviously not completely tamper-proof, but should be good enough to prevent accidental triggers.